.travis.yml

```diff
@@ -69,7 +73,6 @@ script:
 env:
   global:
     - ORGNAME="omnia"
-    - OE_LICENSE="$HOME/oe_license.txt"
```

Comment:
Is there a reason you've removed this?

Reply:
Yes! Because it also appears again on line 78, six lines below. I was just removing the duplicate.